source/common/config/new_grpc_mux_impl.cc

```diff
@@ -91,6 +91,7 @@ void NewGrpcMuxImpl::onDiscoveryResponse(
 
 void NewGrpcMuxImpl::onStreamEstablished() {
   for (auto& [type_url, subscription] : subscriptions_) {
+    UNREFERENCED_PARAMETER(type_url);
```

Comment:
type_url -> _

Reply:
There are some examples of UNREFERENCED_PARAMETER(_); in the code already.  I don't think that the rename to _ will suppress the warnings; _ is not special.